torch/_torch_docs.py

```diff
@@ -5278,6 +5278,45 @@ def merge_dicts(*dicts):
             [ 8,  9]])
 """)
 
+add_docstr(torch.nan_to_num,
+           r"""
+nan_to_num(input, nan=0.0, posinf=None, neginf=None, *, out=None) -> Tensor
+
+Replace NaN with zero and infinity with large finite numbers (default behaviour)
```

Comment:
What about this to replace both paragraphs? 

"Replaces \`\`NaN`\`\, positive infinity, and negative infinity values in :attr:`input` with the values specified by :attr:\`nan\`, :attr:\`posinf\`, and :attr:\`neginf\`, respectively. By default, \`\`NaN\`\`s are replaced with zero, positive infinity is replaced with the greatest finite value representable by :attr:\`input\`'s dtype, and negative infinity is replaced with the least finite value representable by :attr:\`input\`'s dtype."